package.json

```diff
@@ -1,101 +1,69 @@
-  },
-  "peerDependencies": {
-    "react": "^16.9.0 || ^17.0.0-0"
+    "types:watch": "tsc --noEmit --watch",
+    "version": "node scripts/version.mjs"
   },
   "devDependencies": {
-    "@babel/cli": "^7.14.3",
-    "@babel/core": "^7.14.3",
-    "@babel/plugin-proposal-class-properties": "^7.13.0",
-    "@babel/plugin-proposal-export-namespace-from": "^7.14.2",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/cli": "^7.14.5",
+    "@babel/core": "^7.14.5",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-export-namespace-from": "^7.14.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/plugin-syntax-import-meta": "^7.10.4",
-    "@babel/plugin-transform-runtime": "^7.14.3",
+    "@babel/plugin-transform-runtime": "^7.14.5",
     "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.14.2",
-    "@babel/preset-flow": "^7.13.13",
-    "@babel/preset-react": "^7.13.13",
-    "@babel/preset-typescript": "^7.13.0",
-    "@babel/register": "^7.13.16",
-    "@babel/runtime": "^7.14.0",
-    "@testing-library/jest-dom": "^5.12.0",
-    "@testing-library/react": "^11.2.7",
-    "@types/babel__core": "^7.1.14",
-    "@types/babel-plugin-tester": "^9.0.3",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-flow": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/preset-typescript": "^7.14.5",
+    "@babel/register": "^7.14.5",
+    "@babel/runtime": "^7.14.5",
+    "@testing-library/jest-dom": "^5.14.1",
     "@types/jest": "^26.0.23",
-    "@types/mini-css-extract-plugin": "^1.4.3",
-    "@types/react": "^16.14.6",
+    "@types/react": "^16.14.8",
     "@types/react-dom": "^16.9.13",
     "@types/testing-library__jest-dom": "^5.14.0",
-    "@types/webpack": "^4.41.28",
-    "@typescript-eslint/eslint-plugin": "^4.24.0",
-    "@typescript-eslint/parser": "^4.24.0",
-    "babel-eslint": "^10.0.2",
-    "babel-jest": "^26.0.1",
+    "@types/webpack": "^4.41.29",
+    "@typescript-eslint/eslint-plugin": "^4.27.0",
+    "@typescript-eslint/parser": "^4.27.0",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.6.3",
     "babel-loader": "^8.2.2",
-    "babel-plugin-tester": "^10.0.0",
     "babel-types": "^6.26.0",
     "chalk": "^4.1.1",
-    "css-loader": "^5.2.4",
-    "eslint": "^7.26.0",
+    "eslint": "^7.28.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-import-resolver-typescript": "^2.4.0",
     "eslint-plugin-flowtype": "^5.7.2",
-    "eslint-plugin-import": "^2.23.2",
+    "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jest": "^24.3.6",
     "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.23.2",
+    "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "flow-bin": "^0.151.0",
     "jest": "^26.6.3",
-    "mini-css-extract-plugin": "^1.6.0",
-    "outdent": "^0.8.0",
-    "prettier": "^2.3.0",
+    "prettier": "^2.3.1",
     "react": "^16.14.0",
```

Comment:
IMO, react should be in `peerDependecies`.

Reply:
It is a peer dependency for the core library: https://github.com/atlassian-labs/react-loosely-lazy/blob/4ea24fac5afb780f95a3cfdceaba21183ac58359/packages/core/react-loosely-lazy/package.json#L36

The root `package.json` dev dependencies are for primarily for any root scripts. In this case, react is there to run the `examples` when running `npm start`.